package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "ember-cli-file-picker",
   "description": "An addon for ember-cli that provides a component to easily add a filepicker to your ember-cli app.",
-  "version": "0.0.10",
+  "version": "0.1.0",
```

Comment:
I wasn't sure how you'd want to increment the version number, so happy to change this if needed.